dev/io.openliberty.microprofile.faulttolerance.4.1.telemetry.metrics.2.0/src/io/openliberty/microprofile/faulttolerance/telemetry/metrics_20/MetricRecorderImpl.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.faulttolerance.telemetry.metrics_20;
+
+import com.ibm.websphere.ras.annotation.Trivial;
+import io.openliberty.microprofile.faulttolerance.telemetry.metrics.integration.AbstractMetricRecorderImpl;
+
+import io.opentelemetry.api.metrics.Meter;
+
+import com.ibm.ws.microprofile.faulttolerance.spi.BulkheadPolicy;
+import com.ibm.ws.microprofile.faulttolerance.spi.CircuitBreakerPolicy;
+import com.ibm.ws.microprofile.faulttolerance.spi.FallbackPolicy;
+import com.ibm.ws.microprofile.faulttolerance.spi.MetricRecorderProvider.AsyncType;
+import com.ibm.ws.microprofile.faulttolerance.spi.RetryPolicy;
+import com.ibm.ws.microprofile.faulttolerance.spi.TimeoutPolicy;
+
+/**
+ * Records metrics using MP Telemetry API 2.0
+ */
+@Trivial
+public class MetricRecorderImpl extends AbstractMetricRecorderImpl {
+
+    public MetricRecorderImpl(String classAndMethod, String metricPrefix, Meter meter, RetryPolicy retryPolicy, CircuitBreakerPolicy circuitBreakerPolicy, TimeoutPolicy timeoutPolicy,
+                              BulkheadPolicy bulkheadPolicy, FallbackPolicy fallbackPolicy, AsyncType isAsync) {
+        super(classAndMethod, metricPrefix, meter, retryPolicy, circuitBreakerPolicy, timeoutPolicy, bulkheadPolicy, fallbackPolicy, isAsync);
+    }
+}
```

Comment:
I don't think we need an abstract class for this any more. The MP Metrics recorders had one to work around changes in the MP Metrics API.

Reply:
This also means we don't need this project, it can all go in one project.